modules/exploits/multi/http/shiro_rememberme_v124_deserialize.rb

```diff
@@ -53,14 +53,15 @@ def initialize(info={})
     )
     register_options(
     [
-      OptString.new('TARGETURI', [ true, 'Base directory path', '/'])
+      OptString.new('TARGETURI', [ true, 'Base directory path', '/']),
+      OptString.new('SHIROKEY', [ true, 'Shiro default key', 'kPH+bIxk5D2deZiIxcaaaA=='])
```

Comment:
Presumably, as this is now configurable, this option represents a `Shiro key` rather than a `Shiro default key` (?)

```suggestion
      OptString.new('SHIROKEY', [ true, 'Shiro key', 'kPH+bIxk5D2deZiIxcaaaA=='])
```

Better yet, describe what the key does:

```suggestion
      OptString.new('SHIROKEY', [ true, 'Shiro encryption key', 'kPH+bIxk5D2deZiIxcaaaA=='])
```

Reply:
I would maybe even go a step further and just call it `ENC_KEY` to reuse an option name already found in some modules:
```suggestion
      OptString.new('ENC_KEY', [ true, 'Shiro encryption key', 'kPH+bIxk5D2deZiIxcaaaA=='])
```

There are a lot of options that get unique names making it sometimes difficult to select possible loot collected that may be usable with various modules.  The context that the key is for `shiro` is built into the module context.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/14522/commits/24e8aeffe58bd0630ef39be6928cbd6f3a952a9c